syntax = "proto3";
package midend;

message TensorShapeDef {
  repeated int64 dim = 1;
}
